packages/components/tooltip/src/tooltip.ts

```diff
@@ -62,7 +65,9 @@ export class Tooltip extends LitElement {
     }
   };
 
-  /** Tooltip position. */
+  /** Tooltip position.
+   * @type {'top' | 'right' | 'bottom' | 'left' | 'top-start' | 'top-end' | 'right-start' | 'right-end' | 'bottom-start' | 'bottom-end' | 'left-start' | 'left-end'}
```

Comment:
I thought we no longer needed to do this? cem will automatically extract this info?

Reply:
Sometimes it's working, sometimes not, that's why I added it here, because it was not working in this case